script.js

```diff
@@ -0,0 +1,8 @@
+menu.onclick = function myFunction() {
+  const x = document.getElementById('myTopnav');
+  if (x.className === "topnav") {
+    x.className += " responsive";
+  } else {
+    x.className = "topnav";
+  }
+}
```

Comment:
semicolon is required here

Reply:
an empty line is required. Just add it at the end of every js file.
https://eslint.org/docs/rules/eol-last